ci/docker/install/requirements

```diff
@@ -24,16 +24,19 @@ requests>=2.20.0,<3
 graphviz<0.9.0,>=0.8.1
 
 # Optional dependencies
-onnx==1.5.0
+onnx==1.5.0;python_version<"3.8"
+onnx==1.7.0;python_version=="3.8"
 # protobuf version frozen due to ps-lite
 protobuf==3.5.2
 scipy==1.4.1
 tabulate==0.7.5
-Cython==0.29.7
+Cython==0.29.7;python_version<"3.8"
+Cython==0.29.19;python_version=="3.8"
 
 # Development dependencies
 cpplint==1.3.0
-pylint==2.3.1  # pylint and astroid need to be aligned
+pylint==2.3.1;python_version<"3.8"
+pylint==2.4.4;python_version=="3.8"  # pylint and astroid need to be aligned
 astroid==2.3.3  # pylint and astroid need to be aligned
```

Comment:
I suspect your lint errors are due to not aligning the pylint and astroid version.